### ISSUE TITLE
Rasa X Server install help- 'please fill all the fields'

### ISSUE THREAD

**Issue by WR1171** [RasaHQ/rasa#3577]
**Rasa version**:
1.0.1
 0.19.0 

**Python version**: 
3.4
**Operating system** (windows, osx, ...):
Linux - Ubuntu 16.04 also tried 18.04 (DigitalOcean)
**Issue**:
I followed the server installations, running on DigitalOcean, met min requirements, installed & my IP address shows Rasa, but…

    when I get to step 4 admin password -

`sudo python rasa_x_commands.py set_password PASSWORD`

I get an error, invalid choice: ‘set_password’ (choose from u’create’ u’delete) So I was able to create an admin username password with -

`sudo python rasa_x_commands.py create admin USERNAME PASSWORD`

It worked, it creates a user, but since this is the free and not enterprise I don’t think I have a username

& my ip address shows an error 'please fill all the fields' can't enter a password, can't clink on enterprise

tl;dr But the big issue, my IP address shows the login screen with Welcome, password sign in or I have enterprise ID, and no matter what I do I get Please full all the fields in red above, can’t click buttons or do anything, I tried Chrome and Firefox, same issue

**Content of configuration file (config.yml)**:
```yml

```

**Content of domain file (domain.yml)** (if used & relevant):
```yaml

```

**Comment by akelad** [comment]
Thanks for raising this issue, @wochinge  will get back to you about it soon.

**Comment by wochinge** [comment]
@WR1171 It's currently a bug that the user password is not shown in the logs when you start the community edition as docker containers.

Can you please try to update the password of the `me` user as a workaround?
Should be `sudo python rasa_x_commands.py create --update admin me <your password>`

**Comment by ftsamis** [comment]
Facing the same issue. It seems that the CE docker image does not create a `me` user at all.
```
# python rasa_x_commands.py create --update admin me password
User me does not exist. To create a new user please run `sudo python rasa_x_commands.py create me <pw> admin`
```

Although it is possible to create the `me` user with:

```
python rasa_x_commands.py create admin me password
```

(and not `rasa_x_commands.py create me <pw> admin` as the output message states)

this apparently triggers the enterprise edition functionality and Rasa X starts hitting URLs that do not exist (in the CE), such as `/api/conversations/me` which respond with `HTTP 404`

**Comment by WR1171** [comment]
Got the same answer as ftsamis

It wants me to create the 'me' user, and I am still running into my landing page tell me 'please fill in all the fields'

ftsamis, are you able to get beyond the login page at all?

http://178.128.185.79

That is my current digital ocean docker install

**Comment by ftsamis** [comment]
@WR1171 You can get beyond the login page by passing the username and the password as GET parameters (which is generally a bad practice). I discovered that when attempting to install Rasa X through pip (which worked btw). In the docker version, even if you manually create a `me` user and successfully login, the actual Rasa X functionality does not work - you'll get a ton of HTTP `404` and `500` status codes.

`http://<url>/login?username=me&password=pass`

**Comment by WR1171** [comment]
Gotcha, well hopefully they can fix the docker setup soon. I'm pretty
stoked to get this thing up and running.  I haven't played with it rasa
before and I should just get it installed locally, but using digital ocean
has become a habit, I like my local computer free of other things running
while I'm multitasking and the ability to refresh a server after I
completely destroy it is great.  I'll keep checking on the updates, thanks
for following up

**Comment by WR1171** [comment]
> 
> 
> @WR1171 You can get beyond the login page by passing the username and the password as GET parameters (which is generally a bad practice). I discovered that when attempting to install Rasa X through pip (which worked btw). In the docker version, even if you manually create a `me` user and successfully login, the actual Rasa X functionality does not work - you'll get a ton of HTTP `404` and `500` status codes.
> 
> `http://<url>/login?username=me&password=pass`

When you installed through PIP were you still running into errors?

**Comment by wochinge** [comment]
@WR1171 @ftsamis I'm looking into this as part of the sprint, so we can hopefully ship a fix with the new release. I'm very sorry about this!

**Comment by ftsamis** [comment]
@WR1171 Installing through pip3 worked fine. Rasa X creates a user and a random password on startup and provides you with the login URL.

**Comment by tmbo** [comment]
will be fixed by https://github.com/RasaHQ/rasa-x/issues/746

**Comment by TyDunn** [comment]
@WR1171 @ftsamis This was fixed in the latest version of Rasa X: `Release 0.19.1`

**Comment by akelad** [comment]
will close this then, let us know if you do encounter it again though

**Comment by zmalq** [comment]
I am getting the error. python not found
![image](https://user-images.githubusercontent.com/11310193/92606252-60d65500-f2d0-11ea-94be-962a5288f2ad.png)

**Comment by akelad** [comment]
can you try `python3` instead?

**Comment by zmalq** [comment]
> can you try `python3` instead?

Yes it worked. Thanks!

**Comment by konandavy** [comment]
It worked for me but when i click on the external link and enter the code it says wrong credentials

**Comment by wochinge** [comment]
@konandavy Can you please create a new issue and provide some information about your deployment like OS and Rasa X version?

**Comment by konandavy** [comment]
ok no problem
